src/openzaak/components/besluiten/openapi.yaml

```diff
@@ -1136,7 +1136,6 @@ components:
           items:
             $ref: '#/components/schemas/FieldValidationError'
     Wijzigingen:
-      title: Wijzigingen
```

Comment:
I don't really consider this a problem, but we should check if we can bring this back.

Reply:
it's not worth the effort